core/TIBCO.Software.changes.txt

```diff
@@ -38,4 +38,9 @@ Modified files:
 com/lowagie/text/pdf/OcspClientBouncyCastle.java
 com/lowagie/text/pdf/PdfPKCS7.java
 com/lowagie/text/pdf/PdfPublicKeySecurityHandler.java
-com/lowagie/text/pdf/PdfReader.java
\ No newline at end of file
+com/lowagie/text/pdf/PdfReader.java
+
+#7 Upgrade Bouncy Castle dependencies to jdk15on-1.62
+Date: July 3, 2019
+Modified files:
+com/lowagie/text/pdf/PdfPublicKeySecurityHandler
```

Comment:
should we add a description of the see changes here?

Reply:
I have written about the See-chnages in the README.md that 

Seeburger changes include making it compatible with newer versions of the BouncyCastle cryptography provider.

I will make a more detailed change log. The README.md is also included in the JAR that is built.